code_samples/data_migration/config/custom_services.yaml

```
@@ -15,3 +15,12 @@ services:
     App\Migrations\Matcher\SectionIdentifierGenerator:
         tags:
             - { name: 'ibexa.migrations.generator.criterion_generator.content' }
+
+    App\Migrations\Step\ReplaceNameStepNormalizer:
+        tags:
+            - 'ibexa.migrations.serializer.step_normalizer'
+            - { name: 'ibexa.migrations.serializer.normalizer', priority: 9000 }
```

This was a joke, priority can be 0. All our normalizers have negative priority, so user defined go first unless they also have lower priority.

```suggestion
            - 'ibexa.migrations.serializer.normalizer'
```